modules/@apostrophecms/piece-type/ui/apos/components/AposPiecesManagerToolbar.vue

```diff
@@ -25,7 +25,8 @@
         @input="filter"
       />
       <AposInputString
-        @input="search" :field="searchField.field"
+        @input="search" @return="search($event, true)"
```

Comment:
Here's where we set the `force` argument to trigger search regardless of the number of characters entered.